SPECS/azurelinux-rpm-macros/azurelinux-rpm-macros.signatures.json

```diff
@@ -14,7 +14,7 @@
   "generate-package-note.py": "bd76a8e88a1356fed74863c38e5cf6a20c1c26426ac94ba21dd172578e8ca2a2",
   "gpgverify": "db0e050f56b694497d70603a6f5c17dd60ddbcf7cee670616851cd389f6767c4",
   "import_all_modules.py": "543d8c90e7db902e3b4587b7c488419e2f20a6d8b34174b7303dcd70794d8fc7",
-  "macros": "348741813b676ad05698e1bb0c52be7940a7b3e976a5a77be0bdeb479e938868",
+  "macros": "29ab8257de3c66fcf5049becc838c61797937f6249f7412890370876e773ef20",
```

Comment:
Please also bump the `Release` tag in the spec. I believe that's mandatory since the public preview.

Reply:
good point, I missed that